package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "request-reply",
-  "version": "1.2.0",
+  "version": "1.2.1",
```

Comment:
When you bump the version in package.json, there is a corresponding value that must change in the package-lock.json file. Running `npm install` should trigger that needed change.